src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/VM.java

```diff
@@ -953,20 +946,20 @@ public boolean isSharingEnabled() {
   }
 
   public boolean isCompressedOopsEnabled() {
-    if (compressedOopsEnabled == null) {
+      //if (compressedOopsEnabled == null) {
```

Comment:
I am not sure I understand how this changes are related to the issue.

Reply:
Sorry, I was working on another CR at the same time and had added debugging code. I thought I stripped it all out before doing the commit, but it looks like I didn't save the changes.

Reply:
Should be fixed now.